build/lib/ios.js

```diff
@@ -27,6 +29,17 @@ class IOS {
 		this.timestamp = options.timestamp;
 	}
 
+	babelOptions() {
+		// eslint-disable-next-line security/detect-non-literal-require
+		const minSupportedIosSdk = version.parseMin(require(path.join(ROOT_DIR, 'iphone', 'package.json')).vendorDependencies['ios sdk']);
```

Comment:
This'll need updating to use `minIosVersion` per #11091/[TIMOB-27292](https://jira.appcelerator.org/browse/TIMOB-27292)